common-arcana.lic

```diff
@@ -51,6 +59,29 @@ module DRCA
     ['Your body is willing', 'You have not recovered'].include?(result)
   end
 
+  def start_barb_abilities(skills, settings)
+    skills
+      .each do |name|
+        next if DRSpells.active_spells[name]
+        activate_barb_buff(name)
+        waitrt?
+        pause 6 if get_data('spells').barb_abilities[name]['type'].eql? 'meditation'
+      end
+  end
+
+  def activate_barb_buff(name)
+    case bput(ability_data['start_command'], ability_data['activated_message'], 'But you are already', 'Your inner fire lacks', 'find yourself lacking the inner fire', 'You should stand', 'You must be sitting')
```

Comment:
I'm pretty sure this `bput` needs to be `DRC.bput`, could you double check that?

Reply:
It does! But my question is how's it working without? It's just *smart*?